content/introduction/getting-started.md

```diff
@@ -553,14 +620,14 @@ Let's rewrite our template to implement these requirements:
 ```
 
 If we run our feature test now, we'll see it fails — because our controller
-action does not actually [_expose_](/guides/1.2/actions/exposures) the books to our view. We can write a test for
+action does not actually [_expose_](/guides/1.1/actions/exposures) the books to our view. We can write a test for
```

Comment:
`/guides/1.1/actions/exposures` to `/actions/exposures` ?

Reply:
Fixed by https://github.com/hanami/guides/pull/3/commits/d5ecb556e4e4f8670ca8f690535049aa435436cd